proto/prometheus/remote/remote.proto

```diff
@@ -14,7 +14,7 @@
 syntax = "proto3";
 package prometheus;
 
-option go_package = "prompb";
+option go_package = "github.com/GreptimeTeam/greptime-proto/go/prometheus/remote";
```

Comment:
Is it needed to update the License comment in `prometheus/remote/*.proto`, appending Greptime Team? like: 

```
Copyright 2016 Prometheus Team
Copyright 2023 Greptime Team
```

Reply:
I add the comments to describe our modification.